diff -ur dictd-1.9.10.orig/configure.in dictd-1.9.10/configure.in
--- dictd-1.9.10.orig/configure.in	Thu Jul 24 15:46:44 2003
+++ dictd-1.9.10/configure.in	Thu Jul 24 15:48:00 2003
@@ -48,14 +48,6 @@
 
 DICT_VERSION=VERSION
 
-AC_ARG_WITH(cc,
-[  --with-cc               with cc (instead of gcc)],
-[
-if test "x${withval}" = xyes; then
-    CC=cc
-fi
-])
-
 AC_CANONICAL_HOST
 AC_PROG_CC
 dnl AC_PROG_CXX
@@ -64,7 +56,6 @@
 
 REALCC="$CC"
 if test "$CC" = gcc; then
-    CFLAGS='-g -O'
     WCFLAGS='-Wall'
     case "$host" in
       dnl did include: -Winline
@@ -76,8 +67,6 @@
     esac
 				# Silence flex/bison output
     SCFLAGS="-Wno-implicit -Wno-unused"
-else
-    CFLAGS='-g'
 fi
 
 local_dmalloc=0
@@ -85,10 +74,6 @@
 local_zlib=0
 local_regex=0
 
-AC_ARG_WITH(cflags,
-[  --with-cflags=FLAGS     use FLAGS for CFLAGS],
-CFLAGS="$withval")
-
 AC_ARG_WITH(prof,
 [  --with-prof             with prof profiling],
 [
Only in dictd-1.9.10: configure.in~
diff -ur dictd-1.9.10.orig/libmaa/configure.in dictd-1.9.10/libmaa/configure.in
--- dictd-1.9.10.orig/libmaa/configure.in	Thu Jul 24 15:46:44 2003
+++ dictd-1.9.10/libmaa/configure.in	Thu Jul 24 15:49:01 2003
@@ -44,14 +44,6 @@
 echo Configuring for libmaa $MAA_MAJOR.$MAA_MINOR
 echo .
 
-AC_ARG_WITH(cc,
-[  --with-cc               with cc (instead of gcc)],
-[
-if test "x${withval}" = xyes; then
-    CC=cc
-fi
-])
-
 AC_CANONICAL_HOST
 AC_PROG_AWK
 AC_PROG_CC
@@ -59,10 +51,7 @@
 
 REALCC="$CC"
 if test "$CC" = gcc; then
-    CFLAGS='-g -O'
     WCFLAGS='-Wall -pedantic'
-else
-    CFLAGS='-g'
 fi
 
 needs_xmalloc=1
@@ -77,14 +66,6 @@
     WCFLAGS="$WCFLAGS -pipe";;
 esac
 
-AC_ARG_WITH(cflags,
-[  --with-cflags=FLAGS     use FLAGS for CFLAGS],
-[
-if test "x${withval}" = xyes; then
-    CFLAGS="$withval"
-fi
-])
-
 AC_ARG_WITH(prof,
 [  --with-prof             with prof profiling],
 [
Only in dictd-1.9.10/libmaa: configure.in~
