diff -Nur dictd-1.5.5.orig/configure.in dictd-1.5.5/configure.in
--- dictd-1.5.5.orig/configure.in	Sat Nov 17 20:19:56 2001
+++ dictd-1.5.5/configure.in	Sat Nov 17 20:20:57 2001
@@ -42,21 +42,12 @@
 echo Configuring for dict
 echo .
 
-AC_ARG_WITH(cc,
-[  --with-cc               with cc (instead of gcc)],
-[
-if test "x${withval}" = xyes; then
-    CC=cc
-fi
-])
-
 AC_CANONICAL_HOST
 AC_PROG_CC
 AC_ISC_POSIX
 
 REALCC="$CC"
 if test "$CC" = gcc; then
-    CFLAGS='-g -O'
     WCFLAGS='-Wall'
     case "$host" in
       dnl did include: -Winline
@@ -68,18 +59,12 @@
     esac
 				# Silence flex/bison output
     SCFLAGS="-Wno-implicit -Wno-unused"
-else
-    CFLAGS='-g'
 fi
 
 local_dmalloc=0
 local_libmaa=0
 local_zlib=0
 
-AC_ARG_WITH(cflags,
-[  --with-cflags=FLAGS     use FLAGS for CFLAGS],
-CFLAGS="$withval")
-
 AC_ARG_WITH(prof,
 [  --with-prof             with prof profiling],
 [
diff -Nur dictd-1.5.5.orig/libmaa/configure.in dictd-1.5.5/libmaa/configure.in
--- dictd-1.5.5.orig/libmaa/configure.in	Mon Jul  6 00:14:34 1998
+++ dictd-1.5.5/libmaa/configure.in	Sat Nov 17 20:21:30 2001
@@ -43,24 +43,13 @@
 echo Configuring for libmaa $MAA_MAJOR.$MAA_MINOR
 echo .
 
-AC_ARG_WITH(cc,
-[  --with-cc               with cc (instead of gcc)],
-[
-if test "x${withval}" = xyes; then
-    CC=cc
-fi
-])
-
 AC_CANONICAL_HOST
 AC_PROG_CC
 AC_ISC_POSIX
 
 REALCC="$CC"
 if test "$CC" = gcc; then
-    CFLAGS='-g -O'
     WCFLAGS='-Wall -pedantic'
-else
-    CFLAGS='-g'
 fi
 
 needs_xmalloc=1
@@ -75,14 +64,6 @@
     WCFLAGS="$WCFLAGS -pipe";;
 esac
 
-AC_ARG_WITH(cflags,
-[  --with-cflags=FLAGS     use FLAGS for CFLAGS],
-[
-if test "x${withval}" = xyes; then
-    CFLAGS="$withval"
-fi
-])
-
 AC_ARG_WITH(prof,
 [  --with-prof             with prof profiling],
 [
