diff -ur dictd-1.9.1/configure.in dictd-1.9.1.new/configure.in
--- dictd-1.9.1/configure.in	Wed Dec 25 20:17:53 2002
+++ dictd-1.9.1.new/configure.in	Wed Dec 25 20:17:04 2002
@@ -249,7 +249,9 @@
 fi
 
 AC_CHECK_LIB(socket,main,REALLIBS="$REALLIBS -lsocket")
-AC_CHECK_LIB(nsl,main,REALLIBS="$REALLIBS -lnsl")
+AC_CHECK_FUNC(gethostbyname, [],
+       AC_CHECK_LIB(nsl, gethostbyname, REALLIBS="$REALLIBS -lnsl")
+)
 
 if test "$local_dmalloc" = 0 -a "$needs_dmalloc" = 1; then
     AC_CHECK_LIB(dmalloc,main,
