--- dictd-1.8.0/configure.in~	Tue Sep 17 15:16:37 2002
+++ dictd-1.8.0/configure.in	Tue Sep 17 15:18:13 2002
@@ -209,7 +209,9 @@
 fi
 
 AC_CHECK_LIB(socket,main,REALLIBS="$REALLIBS -lsocket")
-AC_CHECK_LIB(nsl,main,REALLIBS="$REALLIBS -lnsl")
+AC_CHECK_FUNC(gethostbyname, [],
+	AC_CHECK_LIB(nsl, gethostbyname, REALLIBS="$REALLIBS -lnsl")
+)
 AC_CHECK_LIB(ltdl,lt_dlopen,REALLIBS="$REALLIBS -lltdl")
 
 if test "$local_dmalloc" = 0 -a "$needs_dmalloc" = 1; then
